docker-swarm-cluster/azuredeploy.json

```diff
@@ -0,0 +1,256 @@
+{
+    "$schema": "https://schema.management.azure.com/schemas/2015-01-01/deploymentTemplate.json#",
+    "contentVersion": "1.0.0.0",
+    "parameters": {
+        "location": {
+            "type": "string",
+            "defaultValue": "West Europe",
+            "allowedValues": [ "East US", "West US", "West Europe", "East Asia", "South East Asia" ],
+            "metadata": {
+                "description": "This is the location where the cluster will be deployed"
+            }
+        },
+        "storageAccountName": {
+            "type": "string",
+            "defaultValue": "dockerswarmstorage",
```

Comment:
Best practice: Please remove default value for storage accounts as this needs to be globally unique and should be a user input